srcpkgs/glances/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'glances'
 pkgname=glances
-version=3.1.7
+version=3.2.2
 revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3-setuptools python3-future python3-psutil"
-checkdepends="python3-psutil"
+depends="python3-setuptools python3-future python3-psutil python3-defusedxml"
```

Comment:
is python3-setuptools really needed? could you check?

Reply:
Indeed, not needed. But `python3-future` pulls it in anyways.